clientSide/src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,41 @@
+.navigation{
+    display: flex;
+}
+
+.navigation  >  p{
+    margin-left: 110px;
+    font-size: 18px;
+}
+
+#home{
+    margin-left: 70px;
+    margin-top: 50px;
+}
+
+#home:hover{
+    text-decoration: none;
+    color: red;
+}
+
+Link:hover{
+    color: red;
+}
+
+#categary{
```

Comment:
Please maintain one class for repeated properties.

Reply:
I changed please chek it once